src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -230,9 +230,9 @@
         },
         "auto_detect_indentation": {
             "type": "boolean",
-            "default": false,
-            "title": "Auto-detect indentation in files",
-            "description": "Whether to automatically detect indentation settings from file contents."
+            "default": true,
+            "title": "Autodetect indentation in files",
+            "description": "When enabled, the indentation for documents not part of an RStudio project will be automatically detected."
```

Comment:
Why are these three lines changed? These don't match what is currently in `main` (maybe this got synced with an older state, or branch, or something?)